_posts/2021-04-20-jump_dev_2021.md

```diff
@@ -0,0 +1,69 @@
+---
+layout: page
+title:  "JuMP-dev 2021"
+date:   2021-04-20
+categories: [jump-dev]
+permalink: /meetings/juliacon2021/
+---
+
+We are pleased to announce that JuMP-dev 2021 will be co-located with
+[JuliaCon 2021](https://juliacon.org/2021/)!
+
+This year JuliaCon is virtual, and will be held 28-30 July, 2021.
+
+The purpose of JuMP-dev is to bring together students, researchers, and
+practitioners with interests in the software aspects of
+[JuMP](https://github.com/jump-dev/JuMP.jl) and related packages. In particular,
+we invite new contributors and those who have not met the core development team.
+
+### How to attend
+
+JuliaCon 2021 is entirely virtual and free to attend, but
+[registration is required](https://juliacon.org/2021/tickets/).
+
+Logistically, JuMP-dev will function as a separate track of JuliaCon. By
+registering for JuliaCon you can attend talks at JuMP-dev and JuliaCon.
+
+All participants will uphold the [JuliaCon Code of Conduct](https://juliacon.org/2021/coc/).
+
+### How to give a talk
+
+Similarly to [previous](/meetings/mit2017) [iterations](/meetings/bordeaux2018)
+of the [workshop](/meetings/santiago2019), we invite participants to present
+work related to JuMP. In particular, we especially seek talks related to:
+
+- JuMP core development ([MathOptInterface](https://github.com/JuliaOpt/MathOptInterface.jl), JuMP 1.0)
+- Mathematical optimization solvers written in Julia
+- Automatic differentiation in Julia (e.g., Cassette, Zygote)
+- Julia interfaces to solvers
+- JuMP extensions (stochastic programming, robust optimization, multiobjective optimization, ...)
+- Optimization libraries that use JuMP
+- Developer tools for JuMP
+- Commercial use of JuMP
+- JuMP for teaching
+
+Speakers are encouraged to highlight challenges and lessons learned, not only
+successes. Talks should be aimed at a general audience, but familiarity with
+JuMP/Julia can be assumed.
+
+See the [previous](/meetings/mit2017/) [JuMP-dev](/meetings/bordeaux2018/)
+[workshops](/meetings/santiago2019) for examples of accepted talks.
+
+Use the following form to submit an abstract. If you have already submitted a
+talk to JuliaCon and would like to speak in the JuMP track instead, please
+submit an abstract and let us know in the "Additional comments" section so we
+can take it into consideration when scheduling talks. There is no guarantee that
+you will be able to switch.
```

Comment:
Would someone be allowed to give two talks, one in both JuliaCon mainline and JuMP-dev? Or is switching the only recourse for those who already have submitted JuliaCon talks.